### PR TITLE
(curl) migrated from ferventcoder

### LINKED ISSUE #697: (curl) Migrate from ferventcoder
Currently outdated and among manual packages

### PR DESCRIPTION
**Pull request by majkinetor**
closes #697

### CODE REVIEW

**Comment by AdmiringWorm** [comment]
just added the chocolatey user, I don't have time to review the PR right now (also the appveyor build fails during au updating).

/CC @majkinetor

**Comment by majkinetor** [comment]
There is no need to review it. Fails for network issue.

**Comment by majkinetor** [comment]
Also, you can always review it later if you want :)

File: automatic/curl/README.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# <img src=""></img> [curl](https://chocolatey.com/packages/curl)
```
Comment (AdmiringWorm) [inline automatic/curl/README.md]: Ah, I see you noticed I had updated the `Update-IconUrl` script to also replace the image url in readme files.
👍

Comment (majkinetor) [inline automatic/curl/README.md]: Do you handle svg files ?

Comment (AdmiringWorm) [inline automatic/curl/README.md]: yup, svg, png, jpg and ico (The last two was added since a package or two used them).

Comment (AdmiringWorm) [inline automatic/curl/README.md]: or are you referring to the newly added optimize function?
Then no, only jpg and png files.

File: automatic/curl/curl.nuspec [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    <tags>curl URL ssh ssl download foss cros-platform cli</tags>
+    <summary>cUrL is a command line tool and library for transferring data with URLs</summary>
+    <description>
+curl is used in command lines or scripts to transfer data. It is also used in cars, television sets, routers, printers, audio equipment, mobile phones, tablets, settop boxes, media players and is the internet transfer backbone for thousands of software applications affecting billions of humans daily.
+
+## Features
+
+Supports DICT, FILE, FTP, FTPS, Gopher, HTTP, HTTPS, IMAP, IMAPS, LDAP, LDAPS, POP3, POP3S, RTMP, RTSP, SCP, SFTP, SMB, SMTP, SMTPS, Telnet and TFTP. curl supports SSL certificates, HTTP POST, HTTP PUT, FTP uploading, HTTP form based upload, proxies, HTTP/2, cookies, user+password authentication (Basic, Plain, Digest, CRAM-MD5, NTLM, Negotiate and Kerberos), file transfer resume, proxy tunneling and more.
+</description>
+    <releaseNotes></releaseNotes>
```
Comment (AdmiringWorm) [inline automatic/curl/curl.nuspec]: empty elements should be removed IMO.

File: automatic/curl/update.ps1 [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+   @{
+        ".\legal\VERIFICATION.txt" = @{
+          "(?i)(\s+x32:).*"            = "`${1} $($Latest.URL32)"
+          "(?i)(\s+x64:).*"            = "`${1} $($Latest.URL64)"
+          "(?i)(checksum32:).*"        = "`${1} $($Latest.Checksum32)"
+          "(?i)(checksum64:).*"        = "`${1} $($Latest.Checksum64)"
+        }
+
+        "$($Latest.PackageName).nuspec" = @{
+            "(\<releaseNotes\>).*?(\</releaseNotes\>)" = "`${1}$($Latest.ReleaseNotes)`$2"
```
Comment (AdmiringWorm) [inline automatic/curl/update.ps1]: Where do you get the ReleaseNotes parameter?
I can't see that you specify it anywhere.

File: automatic/curl/README.md (hunk 1 above)
Comment (majkinetor) [inline automatic/curl/README.md]: I am refering to nuspec icon url (and README.md)

Comment (AdmiringWorm) [inline automatic/curl/README.md]: then yes
https://github.com/chocolatey/chocolatey-coreteampackages/blob/master/scripts/Update-IconUrl.ps1#L114-L119

Comment (AdmiringWorm) [inline automatic/curl/README.md]: however, the Readme.md file wont get updated if the nuspec file is already updated with the latest icon url

File: automatic/curl/update.ps1 (hunk 3 above)
Comment (majkinetor) [inline automatic/curl/update.ps1]: Accidental ... but perhaps useful for the future...

**Pull request merged**